sirepo/srtime.py

```diff
@@ -44,15 +55,24 @@ def api_adjustTime(days=None):
     """
     from sirepo import http_reply
 
+    assert pkconfig.channel_in_internal_test(), \
+        'You can only adjust time in internal test'
     adjust_time(days)
     return http_reply.gen_json_ok({
         'adjustedNow': utc_now().isoformat(),
         'systemNow': datetime.datetime.utcnow().isoformat(),
     })
 
 
+def init():
+    if not pkconfig.channel_in_internal_test():
```

Comment:
does this have to protect itself from re-entrancy.

Conditional should be inverted:

```py
if pkconfig.channel_in_internal_test():
    return
<do_work>
```

Reply:
I don't believe that it has to protect itself. None of the state that is set could be mucked up by re-entrancy. Agree?

Reply:
I think it's a practical to have it protect itself. There are some inits (init_apis) that are protected by the caller, but otherwise, the function should protect itself even if the values are idempotent. If someone adds something to the init, they might think it is protect.